po/aardwolf/aardwolf.pot

```diff
@@ -180,84 +180,88 @@ msgstr ""
 msgid "Aardwolf Instance"
 msgstr ""
 
-# aardwolf-templates/src/../compiled_templates/templates/template_new_post.rs:28
-msgid "Awoo!"
```

Comment:
You didn't like Awoo? :P

Reply:
the gettext macros have some issues (they're not deterministic)

Reply:
that'll be something to look into with @BaptisteGelez